fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/populator/AbstractWorkbookPopulator.java

```diff
@@ -66,9 +66,9 @@ protected void writeDate(int colIndex, Row row, String value, CellStyle dateCell
         try {
             SimpleDateFormat formatinDB = null;
             if (value.matches("\\d{4}-\\d{1,2}-\\d{1,2}")) {
-                formatinDB = new SimpleDateFormat("yyyy-mm-dd");
+                formatinDB = new SimpleDateFormat("yyyy-MM-dd");
```

Comment:
https://docs.oracle.com/javase/7/docs/api/java/text/SimpleDateFormat.html:

* m = Minute in hour
* M = Month in year